f2/src/forms/WhenDidItHappenForm.js

```diff
@@ -76,68 +46,53 @@ export const WhenDidItHappenForm = (props) => {
     values.description = ''
   }
 
-  const formatData = (values) => {
-    let errors = {}
-
+  const clearData = (values) => {
     if (values.incidentFrequency === 'once') {
-      const happenedOnceError = evalDate(
-        values.happenedOnceDay,
-        values.happenedOnceMonth,
-        values.happenedOnceYear,
-      )
-
-      if (happenedOnceError) {
-        errors['happenedOnceError'] = happenedOnceError
-      } else {
-        clearDateRange(values)
-        clearDescription(values)
-      }
+      clearDateRange(values)
+      clearDescription(values)
     } else if (values.incidentFrequency === 'moreThanOnce') {
-      const dateRangeErrors = evalDateRange(values)
-
-      if (dateRangeErrors.startError) {
-        errors['startError'] = dateRangeErrors.startError
-      }
-
-      if (dateRangeErrors.endError) {
-        errors['endError'] = dateRangeErrors.endError
-      }
       clearHappenedOnce(values)
       clearDescription(values)
     } else {
       clearHappenedOnce(values)
       clearDateRange(values)
     }
-
-    return errors
   }
 
   return (
     <React.Fragment>
       <Formik
         initialValues={whenDidItHappen}
-        initialStatus={{ errors: '' }}
-        validationSchema={whenDidItHappenFormSchema()}
         validateOnChange={false}
```

Comment:
Do we still need validateOnChange={false}

Reply:
You're right, this option is no longer needed.